primitives/src/projects.rs

```diff
@@ -93,7 +91,7 @@ where
 /// Default status - storage req
 impl Default for Status {
 	fn default() -> Self {
-		Status::Proposed
```

Comment:
hi, we're no longer changing the default because of more discussion on the specs, and since seeding is no longer done onchain